cms/src/main/java/uk/nhs/digital/arc/json/BasicBodyItem.java

```diff
@@ -0,0 +1,24 @@
+package uk.nhs.digital.arc.json;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BasicBodyItem extends ReferencableUrlElement {
+
+    @JsonIgnore
+    protected Map<String, Object> additionalProperties = new HashMap<String, Object>();
```

Comment:
You can reduce the verbosity of this map to `protected Map<String, Object> additionalProperties = new HashMap<>();` as the compiler will infer the type